src/cpl/nuopc/atm_import_export.F90

```diff
@@ -334,14 +455,12 @@ subroutine import_fields( gcomp, cam_in, restart_init, rc)
 
     ! -----------------------------------------------------
     ! Set field pointers in import state and
-    ! copy from field pointer to chunk array data structure
+    ! copy from field pointer to CAM-SIMA array data structure
     ! -----------------------------------------------------
 
-    use spmd_utils        , only : masterproc
     use camsrfexch        , only : cam_in_t
-    use physics_grid      , only : columns_on_task
     use shr_const_mod     , only : shr_const_stebol
-!CAMDEN TODO Need to uncomment these once carbon cycling is enabled in CAMDEN:
+!CAM-SIMA NOTE: Need to uncomment these once carbon cycling is enabled in SIMA.
```

Comment:
I am pretty confident this will be caught as soon as anyone tries to do anything with an interactive carbon cycle in CAM-SIMA.  It's also unclear to me if that whole methodology will change in the future, so I am fine to just leave it as-is for now (at worst it just means there will be some randomly commented-out code that will need to be removed in the future).  Again if you disagree please let me know!